rest/rest-server/src/test/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRestJobTest.java

```diff
@@ -113,6 +116,29 @@ public void testJobsInfo() throws Throwable {
         RestTestUtils.assertJobsInfoPage(expectedJobs, actualPage);
     }
 
+    @Test
+    public void testJobsInfoList() throws Throwable {
+        List<JobInfo> expectedJobs = new ArrayList<>();
+        JobInfo jobData1 = RestTestUtils.newMockedJobInfo("2", null, 10);
+        expectedJobs.add(jobData1);
+        JobInfo jobData2 = RestTestUtils.newMockedJobInfo("4", null, 2);
+        expectedJobs.add(jobData2);
+        JobInfo jobData3 = RestTestUtils.newMockedJobInfo("10", null, 8);
+        expectedJobs.add(jobData3);
+
+        List<String> jobIds = new ArrayList<>();
+        jobIds.add("2");
+        jobIds.add("4");
+        jobIds.add("10");
+        when(mockOfScheduler.getJobsInfoList(jobIds)).thenReturn(expectedJobs);
+
+        List<UserJobData> actualList = restInterface.jobsInfoList(sessionId, jobIds);
+
+        for (int i = 0; i < expectedJobs.size() && i < actualList.size(); i++) {
+            RestTestUtils.assertJobInfo(expectedJobs.get(i), actualList.get(i));
+        }
```

Comment:
would it be possible to test that a non existent job id throws an exception ?

Reply:
yes, i'll add that

Reply:
I don't think it's possible because the method getJobsInfoList isn't actually called, we mock the call by saying what it should return.